app/src/main/java/ingsw/group1/findmyphone/activity/CreateContactActivity.java

```diff
@@ -0,0 +1,52 @@
+package ingsw.group1.findmyphone.activity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import ingsw.group1.findmyphone.R;
+import ingsw.group1.findmyphone.contacts.SMSContactManager;
+import ingsw.group1.msglibrary.SMSPeer;
+
+/**
+ * Activity for add a new contact
+ *
+ * @author Giorgia Bortoletti
+ */
+public class CreateContactActivity extends AppCompatActivity {
+
+    private EditText contactName;
+    private EditText contactPhone;
+    private Button newContactButton;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'newContactButton' could be replaced by a local variable.](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=4929954)